sfa_dash/templates/forms/cdf_forecast_group_form.html

```diff
@@ -44,9 +44,7 @@
 </div>
 <div class="form-element">
   {{ form.select('Interval label', 'interval_label',
-                 {'beginning': 'Beginning',
-                  'ending': 'Ending',
-                  'instant': 'Instant'},
+                 interval_label_options,
                   'Indicates if a time labels the start or the end of an interval.',
```

Comment:
this text should be changed to be more generic or to include instantaneous and event.

Reply:
and be set for all of the forms